packages/shared/src/components/profile/UserMetadata.tsx

```diff
@@ -57,6 +66,30 @@ export function UserMetadata({
           date={new Date(createdAt)}
         />
       </div>
+      {company && (
```

Comment:
Always better to bool cast `!!company` because if `undefined` React can render it as string `'undefined'` sometimes